.github/workflows/continuous-integration.yml

```diff
@@ -22,8 +22,6 @@ env:
 jobs:
   ios-compat:
     runs-on: macos-11
-    env:
-      DEVELOPER_DIR: /Applications/Xcode_12.5.app
```

Comment:
Do we want to do this in this PR -- ideally not, but if it's required, then can we also update generate-docs and release.yml?